ext/osx/puppet.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+                <key>PATH</key>
+                <string>/opt/puppetlabs/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+                <key>RUBYLIB</key>
+		<string>/opt/puppetlabs/puppet/lib/ruby/site_ruby/2.1.0/</string>
```

Comment:
hmm, is there anyway we could make this an erb and process it as we're installing it, so that we don't have to hardcode the ruby version here?